.rubocop_todo.yml

```diff
@@ -32,19 +15,10 @@ Layout/TrailingEmptyLines:
     - 'lib/fhir_models/bootstrap/json.rb'
     - 'lib/fhir_models/r4/fhir_ext/structure_definition.rb'
     - 'lib/fhir_models/r4/fhir_ext/structure_definition_compare.rb'
-    - 'lib/fhir_models/r4/fhir_ext/structure_definition_finding.rb'
     - 'lib/fhir_models/r4b/fhir_ext/structure_definition.rb'
     - 'lib/fhir_models/r4b/fhir_ext/structure_definition_compare.rb'
-    - 'lib/fhir_models/r4b/fhir_ext/structure_definition_finding.rb'
-
-# Offense count: 3
-# Cop supports --auto-correct.
-# Configuration parameters: AllowInHeredoc.
-Layout/TrailingWhitespace:
-  Exclude:
-    - 'lib/fhir_models/fhir_ext/common_structure_definition.rb'
-    - 'lib/fhir_models/fhir_ext/common_structure_definition_compare.rb'
-    - 'lib/fhir_models/fhir_ext/common_structure_definition_finding.rb'
+    - 'lib/fhir_models/r5/fhir_ext/structure_definition.rb'
```

Comment:
Some of these files still don't exist.

Reply:
This is auto generated. Maybe we should remove this file? It does not anything. I manually removed some lines as those are not there.